Automattic/newspack-theme
# Fix up blog posts page styles

### All Submissions:


### Changes proposed in this Pull Request:

Though not a 'featured' area of the theme, the blog posts page -- the default page where all the blog posts display -- should look okay. In this PR I've updated the styles to match the search results, with the featured images floated to the left and displaying only excerpts, not full content.

**Before:**

![image](https://user-images.githubusercontent.com/177561/62145830-fc615d80-b2a8-11e9-87e8-e9ed43b3257b.png)

**After:**

![image](https://user-images.githubusercontent.com/177561/62145771-e2c01600-b2a8-11e9-86ee-cdbd904387a9.png)

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build`. 
2. View the page you've set as the blog posts page under Customizer > Homepage Settings.
3. Confirm that it looks like the search results page. 

### Other information:
